test/dune

```diff
@@ -1,165 +0,0 @@
-(include dune.inc)
-
-; Generates rules
-
-(rule
- (targets dune.inc.gen)
- (deps    (package mdx))
- (action
-  (with-stdout-to %{targets}
-   (progn
-    (run ocaml-mdx rule %{dep:pp.md})
-    (run ocaml-mdx rule %{dep:ellipsis.md})
-    (run ocaml-mdx rule %{dep:envs.md})
-    (run ocaml-mdx rule %{dep:empty_line.md})
-    (run ocaml-mdx rule %{dep:spaces.md})
-    (run ocaml-mdx rule %{dep:errors.md})
-    (run ocaml-mdx rule %{dep:heredoc.md})
-    (run ocaml-mdx rule %{dep:mlt.md})
-    (run ocaml-mdx rule %{dep:semisemi.md})
-    (run ocaml-mdx rule %{dep:exit.md})
-    (run ocaml-mdx rule %{dep:padding.md})
-    (run ocaml-mdx rule %{dep:multilines.md})
-    (run ocaml-mdx rule %{dep:lines.md})
-    (run ocaml-mdx rule %{dep:lwt.md})
-    (run ocaml-mdx rule %{dep:non-det.md})
-    (run ocaml-mdx rule %{dep:code.md})
-    (run ocaml-mdx rule %{dep:set_environment_variable.md})
-    (run ocaml-mdx rule %{dep:unset_environment_variable.md})
-    (run ocaml-mdx rule --prelude-str "#require \"lwt\"" --prelude-str "toto:let x = \"42\"" %{dep:prelude.md})
-    (run ocaml-mdx rule --prelude prelude.ml %{dep:prelude_file.md})
-    (run ocaml-mdx rule --direction=to-ml %{dep:dune_rules.md})
-    (run ocaml-mdx rule %{dep:require/require-package.md})
-   ))))
```

Comment:
That part is gone. It generates the rules for a few tests and include them.
Is it intended?

Reply:
Sorry I missed that you do this from `gen_dune_rules.ml`.